cluster-scope/base/core/namespaces/okd-team/resourcequota.yaml

```
@@ -0,0 +1,12 @@
+kind: ResourceQuota
+apiVersion: v1
+metadata:
+  name: large
```

Sorry, I probably was not clear enough... You're half way there :smile: 

Please rename the resource to `okd-custom`
```suggestion
  name: okd-custom
```